submissions/dyworld-history/dyworld-history.md

```diff
@@ -0,0 +1,49 @@
+
+When I started to mod, there was only 1 major overhaul mod, called F-Mod. I wanted to have the same as that mod, and started. It was clunky in the beginning, but eventually DyTech touched every aspect of the game. I rewrote it several times under somewhat the same name, but the content stayed mostly the same. By rewriting, I learned to code it better, easier to read and edit things. I always had several ideas that were a little out of reach for me, but I always strived to make planets into Factorio since I started modding. Luckily Earendel did that now, and I can work on other projects
+
+The DyTech was very populair back in the day. I even played together a few times with Xterminator and Oninaig while they were doing a lets play on Youtube. After a few years (Factorio 0.15) I started to work on the next step of DyTech, DyWorld. A major rewrite after 2 years of no modding, to fit DyTech better into the ever evolving Factorio. I made great strides, and worked on it till just before Factorio 1.0.0. Several content creators played it on Youtube:
+[KatherineOfSky DyWorld Playthrough](https://www.youtube.com/playlist?list=PL4o6UvJIdPNqubR5oXdx9SqKFoYW_SL-q), 
+[Nexxus Gaming DyWorld Playthrough](https://www.youtube.com/watch?v=Eqs8dakCzL8&list=PLXEOcXLt0r4RxjOgak-BRzO8PMoMnqnoy), 
+[Lumatan DyWorld Playthrough (German)](https://www.youtube.com/watch?v=eH6lu5N-oPQ&list=PLEui1S1GUHOMKnVrLjgybKRKzBNMIHTfk)
+
+About august of last year my itch to mod was back, bigger then before. I decided to rewrite the mod, and add the content of the mod around 1 very specific thing in mind: A story. Thus I rewrote DyWorld into DyWorld-Dynamics.
+But it is now almost a year later, and it is still not released. Well, due to some personal issues I could not do anything a few months. When that was taken care of, I returned to keep working on it. And now it is getting closer to release. Adding in content around a story is much more work then I previously anticipated, but it is fun to do. Luckily I have several beta testers to help me take care of issues while I write everything.
+
+## The Story ##
+
+This is my main focus of DyWorld-Dynamics. I wrote everything with the story elements in mind, but it will work without it as well. What is the story about?
+
+The story starts with crash landing on a desolate planet, filled with weird flora and fauna. You have no idea how you got there and where you are. As the story progresses, you name the planet, find out why you crash landed there, learn about other planets and move there and find an epic conclussion of the story.
```

Comment:
Could you give one more point about the story, to give us some clue as to why the story might be epic?